pxtlib/emitter/backjs.ts

```diff
@@ -127,7 +128,7 @@ switch (step) {
         let info = nodeLocationInfo(proc.action) as FunctionLocationInfo
         info.functionName = proc.getName()
         writeRaw(`${proc.label()}.info = ${JSON.stringify(info)}`)
-
+        writeRaw(`${proc.label()}.continuations = [ ${async_continuations.join(",")} ]`)
```

Comment:
do we need to write this if ``async_continuations`` is empty? (note: js typeically uses camelized variable names)